src/rviz/default_plugin/interactive_markers/interactive_marker.h

```diff
@@ -30,6 +30,18 @@
 #ifndef RVIZ_INTERACTIVE_MARKER_H_
 #define RVIZ_INTERACTIVE_MARKER_H_
 
+// Import/export for windows dll's and visibility for gcc shared libraries.
```

Comment:
Please move this boiler-plate block into an include file.

Reply:
Sure thing!